xc/sitecore/InstallCommercePackages.ps1

```diff
@@ -150,12 +107,9 @@ $node.defaultShopName = $defaultShopName; `
 $xml.Save($pathToConfig);
 
 # Initialize the commerce engine
-$bearerToken = GetIdServerToken -userName $sitecoreUserName -password $sitecorePassword -urlIdentityServerGetToken "${identityServerUrl}connect/token"
+# TODO Enable 
```

Comment:
TODO?

Reply:
Removed TODO and the script in comments.